packages/http/test/test-host.ts

```diff
@@ -81,6 +81,31 @@ export async function compileOperations(
   return [details, diagnostics];
 }
 
+export interface CompileOperationsResult {
+  runner: BasicTestRunner;
+  operations: HttpOperation[];
+  diagnostics: readonly Diagnostic[];
+}
+
+export async function compileOperationsFull(
```

Comment:
isn't that basically the same as `getOperationsWithServiceNamespace`

Reply:
No because I need the runner so that I can pass `program` to `isHeader`.